src/items/class-features/terrifying_blast_(ex).json

```diff
@@ -37,7 +37,7 @@
       "gmnotes": "",
       "short": "",
       "unidentified": "",
-      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p><sub><img src=\"systems/sfrpg/images/cup/gameplay/mind.webp\" alt=\"\" width=\"25\" height=\"25\"></sub><span> </span>Mind-Affecting<sub><img src=\"systems/sfrpg/images/cup/gameplay/sense.webp\" alt=\"\" width=\"25\" height=\"25\"></sub><span> </span>Sense-Dependent</p>\n<p>When you throw a grenade, each creature within the radius of the explosion that fails its Reflex save against the grenade and takes damage from it must succeed at a Will save or gain the @UUID[Compendium.sfrpg.conditions.nywSVEeDvp1bk7L3]{shaken} condition for 1 round. This is a fear effect.</p>"
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p>@Icon[type:mind-affecting] <span> </span>Mind-Affecting@Icon[type:sense-dependent] <span> </span>Sense-Dependent</p>\n<p>When you throw a grenade, each creature within the radius of the explosion that fails its Reflex save against the grenade and takes damage from it must succeed at a Will save or gain the @UUID[Compendium.sfrpg.conditions.nywSVEeDvp1bk7L3]{shaken} condition for 1 round. This is a fear effect.</p>"
```

Comment:
Clear out an the spans, and ensure the icons and labels follow the same format as other features (here they're on the same line, with no space between mind-affecting and the icon for sense dependent)